cypress/e2e/shared/explorer.test.ts

```diff
@@ -437,14 +437,16 @@ describe('DataExplorer', () => {
 
       cy.getByTestID('time-machine--bottom').then(() => {
         cy.getByTestID('flux-editor', {timeout: 30000})
+          .wait(1000) // TODO: find another way to wait on the LSP to be initalized
```

Comment:
this is required otherwise the lsp isn't initialized in time...need to look and see if there is something else we can wait on

Reply:
found that I could do `cy.get('.squiggly-error', {timeout: 30000})` instead